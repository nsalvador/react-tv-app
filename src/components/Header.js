import React from 'react';
import { NavLink } from 'react-router-dom';

const Header = () => (
	<header>
		<h1>React TV App</h1>
		<ul>
			<li>
				<NavLink to="/dashboard" activeClassName="is-active">
					Dashboard
				</NavLink>
			</li>
			<li>
				<NavLink to="/search" activeClassName="is-active">
					Search
				</NavLink>
			</li>
		</ul>
	</header>
);

export default Header;
