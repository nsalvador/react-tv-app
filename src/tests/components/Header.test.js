import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../components/Header';

describe('Component: <Header />', () => {
	it('should successfully render component', () => {
		const wrapper = shallow(<Header />);
		expect(wrapper).toMatchSnapshot();
	});
});
