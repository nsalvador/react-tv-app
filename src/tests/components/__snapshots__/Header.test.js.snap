// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: <Header /> should successfully render component 1`] = `
<header>
  <h2>
    React TV App
  </h2>
  <ul>
    <li>
      <NavLink
        activeClassName="is-active"
        to="/dashboard"
      >
        Dashboard
      </NavLink>
    </li>
    <li>
      <NavLink
        activeClassName="is-active"
        to="/search"
      >
        Search
      </NavLink>
    </li>
  </ul>
</header>
`;
